importer/tests/test_import.py

```diff
@@ -0,0 +1,56 @@
+"""
+Tests for LORE imports.
+"""
+
+# standard library
+from unittest import TestCase
+from os.path import abspath, dirname, exists, join
+from os import mkdir
+
+# PyPi
+import requests
+
+# local
+from importer.api import import_course_from_file
+
+# TODO: Also test this course:
+# https://github.com/pmitros/edX-Insider/archive/master.zip
+def get_course_zip():
+    """
+    Get the path to the demo course.
+    """
+    url = "https://github.com/edx/edx-demo-course/archive/master.zip"
```

Comment:
or a git submodule, but then we all have to do recursive clones.  I believe I mentioned it on the issue, but I think we should actually just create a small course for testing, one that exhibits bad behaviors we have found.  The DevOps course would be perfect for this and is much smaller (1.1M), it just needs to have some sanitizing of it's policy file before becoming open.  In fact, for this test, we really should just be having a minimal course, i.e. a single course.xml file in a tar.gz or zip file, since we should assume (or create the tests to do so, since we know it isn't well tested) the piece that xbundle does is well tested, and we just need to test our code here.